lib/active_record/annotate/file.rb

```diff
@@ -19,39 +20,46 @@ def annotate_with(annotation, configurator)
           # separate magic comments from the annotation with an empty line
           magic_comments << nil
         end
-        
+
         while @lines.first.start_with?('#') || @lines.first.blank?
           # throw out comments and empty lines
           # in the beginning of the file (old annotation)
           @lines.shift
         end
-        
-        if configurator.yard?
-          backticks = '# ```'
-          annotation.unshift(backticks).push(backticks)
+
+        if annotation.first.start_with?("# create_table")
+          if configurator.yard?
+            backticks = '# ```'
+            annotation.unshift(backticks).push(backticks)
+          end
+
+          @lines.unshift(*annotation, nil)
+        elsif configurator.debug
```

Comment:
I think this should be
```suggestion
        elsif configurator.debug?
```